### ISSUE TITLE
How do I hide servers that do not have traffic? on Grafana

### ISSUE THREAD

**Issue by eden-n** [grafana/grafana#2201]
How do I hide servers that do not have traffic.
I do not want to display them on the graph on Grafana?
thanks!

**Comment by eden-n** [comment]
?

**Comment by torkelo** [comment]
@eden-n you do not understand your own question? :) 

In Graphite there are filter function that you can use to filter out series. I see you do not mention what time series store you are using so I am just guessing you use Graphite. 

If the series is all null values it can be hidden using an legend option (under axis & grid tab)

**Comment by eden-n** [comment]
![image](https://cloud.githubusercontent.com/assets/12985048/8276292/18372ee8-18bd-11e5-9445-0decf1162919.png)

Hey Torkelo,
I have attached an image showing a graph in which servers we removed them from our DB, but still displayed in Grfiit. I want to remove them from the system, and remove them from the graphs.
How can I do this?
Thank you!

**Comment by eden-n** [comment]
Hey Torkelo,
I have attached an image showing a graph in which servers we removed them
from our DB, but still displayed in Grfiit. I want to remove them from the
system, and remove them from the graphs.
How can I do this?
Thank you!







2015-06-21 17:21 GMT+03:00 Torkel Ödegaard <notifications@github.com>:

**Comment by torkelo** [comment]
Sorry, I have no idea, since you have not mentioned what DB you are using. 

Grafana gets the list of series from your time series store, it does not look like you removed them.

**Comment by torkelo** [comment]
Can you show me your metric query? What database do you use?  What did you do to remove the old servers from your time series store? An image gives me zero information for how to help you as this needs to be solved in your backend time series store, or in your metric query. 

The only option you have in Grafana to hide series returned by your time series query is if all values in a a series are null: 
![image](https://cloud.githubusercontent.com/assets/10999/8276603/c71c136a-18b8-11e5-821a-fa6ca295c96e.png)

If that is the case check the "hide empty" checkbox

**Comment by eden-n** [comment]
We use the system 1.6.1.
The check  does not show us "hide empty" checkbox
Thanks for the help.

[image: תמונה מוטבעת 1]

[image: תמונה מוטבעת 2]

2015-06-22 9:29 GMT+03:00 Torkel Ödegaard <notifications@github.com>:

**Comment by eden-n** [comment]
What do you suggest?
Are you adding information? Did that information understand the problem?

2015-06-22 9:52 GMT+03:00 Eden Noyman <eden.noy16@gmail.com>:

**Comment by torkelo** [comment]
Sorry, for me to help you, I still require a lot more information (which I have request 3 times). I understand English is not your native language and you seem not to understand what I am writing :( 

You are using a very old version of Grafana. The `hide empty` feature was introduced in Grafana 2.0.  

More info:
1) What database do you use (Graphite or InfluxDB or OpenTSDB) ???
2) What does your metric query look like? 
3) What actions did you perform to remove old metrics?

**Comment by eden-n** [comment]
I'm sorry for my poor language, English is not my native languageץ
About database  We use Graphite
Regarding the second question, I do not understand what you mean.
I did not do any operations before, I tried to search the internet, and
there was no answer to the problem.
I know the old version of graphana. Check with my boss about an upgrade
version.
Thank you very much!

2015-06-22 13:55 GMT+03:00 Torkel Ödegaard <notifications@github.com>:
